frontend/rust-lib/flowy-document2/src/event_handler.rs

```diff
@@ -15,23 +15,14 @@ use flowy_error::{FlowyError, FlowyResult};
 use lib_dispatch::prelude::{data_result_ok, AFPluginData, AFPluginState, DataResult};
 
 use crate::entities::*;
-use crate::{manager::DocumentManager, parser::json::parser::JsonToDocumentParser};
-
-fn upgrade_document(
-  document_manager: AFPluginState<Weak<DocumentManager>>,
-) -> FlowyResult<Arc<DocumentManager>> {
-  let manager = document_manager
-    .upgrade()
-    .ok_or(FlowyError::internal().with_context("The document manager is already dropped"))?;
-  Ok(manager)
-}
+use crate::{manager::DocumentManager, parser::json::parser::JsonToDocumentParser, utils};
 
 // Handler for creating a new document
 pub(crate) async fn create_document_handler(
   data: AFPluginData<CreateDocumentPayloadPB>,
   manager: AFPluginState<Weak<DocumentManager>>,
 ) -> FlowyResult<()> {
-  let manager = upgrade_document(manager)?;
+  let manager = utils::upgrade_document(manager)?;
```

Comment:
I think it doesn't need to move to a new file. The function upgrade_document only use in this file